index.html

```diff
@@ -108,22 +123,23 @@ <h3>I'm a software developer</h3>
         </ul>
       </div>
     </section>
-    <section class="works" id = "portfolio">
+    <section class="works" id="portfolio">
       <div class="recent-works">
-        <div class="title"> 
+        <div class="title">
           <div class="title-h1">
             <h1>My Recent Works</h1>
           </div>
           <div class="line"></div>
         </div>
         <div class="section-2">
-          <div> <img src="img/Placeholder.svg" alt="" /> </div>
-          <div class="multi-post-stories"> 
-             <h2>Multi-Post Stories</h2>
+          <div><img src="img/Placeholder.svg" alt="" /></div>
```

Comment:
I recommend you add a description in the `Alt`  attribute to specifies an alternate text for an area, if the image cannot be displayed.